backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -43,14 +43,45 @@
   }
 }
 
+// Page Layout
+//---------------------------------------------------
+
+html {
+  height: 100%;
+}
+
+body {
+  position: relative;
+  min-height: 100%;
+}
+
+.admin-nav {
+  @include position(fixed, 0 null 0 0);
```

Comment:
Sorry, just pulled you branch and reloaded the browser and this was the first thing I saw :blush: 

No, a CSS only solution won't work. ([Where is `position: sticky`](http://caniuse.com/#feat=css-sticky), if you need it ^_^)

I really love the Wordpress approach, it feels very natural. But this is not affix, right? Because it sticks to bottom as soon as you reach the maximum height of the sidebar. Very neat.